docs/templates/plugin.rst.j2

```diff
@@ -222,7 +222,7 @@ Notes
 Examples
 --------
 
-.. code-block:: yaml+jinja
+.. code-block:: ansible-yaml+jinja
```

Comment:
I wonder if, for the Ansible documentation only, we can make this the default for yaml and yaml+jinja ?
So that we don't have to modify the complete documentation (until this hits upstream and our infra).

I would only move to ansible-yaml+jinja when this is a common thing.

Reply:
I think that should be possible; a simple test should reveal whether this works ;-) I would assume that if you re-use the default lexer names when adding a new lexer, that this one will be used before the built-in ones.

I think a good point for this change is that in case someone accidentally specifies `yaml` or `yaml+jinja` lexing somewhere in the documentations, they get the same lexer than the rest of the documentation.

Reply:
All YAML code-blocks in our current documentation (excluding module/plugin docs) use either *yaml* or *yaml+jinja* and I think it's best to not change it to something that is only known on our infra.

Your changes should probably be upstreamed for both *yaml* and *yaml+jinja*.

But I also see value in having an improved Ansible-extension with knowledge of directives and specific directive-value types.

Reply:
Ah, now I see what you mean. The reason I used `ansible-xxx` was to emphasize that this is not the standard lexer. I didn't meant "this is a lexer with some Ansible-specific stuff".

Reply:
I removed the prefix